suricata-ocsf/package.yaml

```diff
@@ -273,113 +224,87 @@ pipelines:
       // tql2
       subscribe "suricata"
       where @name == "suricata.smb"
-      this = { event: this }
-      tree_uid = string(event.smb.tree_id)
-      if event.smb.filename != null {
-        file = {
-          type_id: 0,
-          name: event.smb.filename,
-          created_time: event.smb.created,
-          modified_time: event.smb.modified,
-          accessed_time: event.smb.accessed,
+      let $activity_id = {
+      FILE_SUPERSEDE: 1,
+      FILE_OPEN: 2,
+      FILE_CREATE: 3,
+      FILE_OPEN_IF: 4,
+      FILE_OVERWRITE: 5,
+      FILE_OVERWRITE_IF:: 6,
+      }
```

Comment:
Throughout the entire file, can you please indent everything that is within braces?

For example:

```
let $activity_id = {
  foo: bar,
  baz: qux,
}
```

```
if condition {
  do_something
} else {
  this = is_indented_as_well
}
```